### ISSUE TITLE
Issue: multiple scans overwrite the zap_scan artifact

### ISSUE THREAD

**Issue by WalterMoar** [zaproxy/action-full-scan#38]
I'm loving the GitHub Action for running ZAP scans, thank you! 💯 

One issue is that when running two scans in parallel they're both creating an artifact called _zap_scan_. So there is a race condition and whichever scan completes first has its artifact overwritten by the scan that finished second. To get around this, after the scan completes I am uploading _report_html.html_ as two differently-named artifacts. However, this still leaves the _zap_scan_ artifact and it is a bit confusing that we have an extra artifact that is essentially a duplicate of one of the other artifacts - although we never know which one. Since artifacts aren't available until after the Action has completed, they cannot be deleted in the Action that created them.

A couple of possibilities are:
1. Have a setting for the name of the artifact (ideal)
1. Have a flag that can turn off the uploading of the _zap_scan_ artifact and users will upload with their own name.

Thank you!

**Comment by thc202** [comment]
The first bullet is planned (e.g. zaproxy/action-baseline#45).

**Comment by WalterMoar** [comment]
Oh, perfect! I didn't realize there was another repo that I should have been looking in. Thank you, looking forward to it.
